models/Class.ts

```diff
@@ -22,7 +23,8 @@ export type ClassCardInfo = {
     image?: string;
     name?: string;
     description: string;
-    ageGroup?: string;
+    borderAge?: number;
+    isAgeMinimal: boolean;
```

Comment:
Could you provide some documentation somewhere about how `borderAge` and `isAgeMinimal` is used? I think maybe in `schema.sql` (or somewhere else if you have a better idea)?